cyclopts/help/help.py

```diff
@@ -192,12 +188,143 @@ def __rich_console__(self, console, options):
+        the member does not exist. This is useful when the list of entries
+        do not have all the same members. This was required for
+        `ColeSpec.render_cell`
+        """
+        if hasattr(self, key):
+            setattr(self, key, value)
+        elif hasattr(self.extras, key):
+            setattr(self.extras, key, value)
+        return self
+
+    def put(self, key: str, value: Optional[Union[str, "LazyData"]]):
+        """Put a attr to the dataclass."""
+        if hasattr(self, key):
+            setattr(self, key, value)
+        elif hasattr(self.extras, key):
+            setattr(self.extras, key, value)
+        else:
+            raise AttributeError(f"'{type(self.extras).__name__}' has no field {key}")
+        return self
+
+    def get(self, key: str, default: Any = None, resolve: bool = False) -> Union["LazyData", "RenderableType"]:
+        if hasattr(self, key):
+            val = getattr(self, key)
+            return _resolve(val, self) if resolve else val
+        if hasattr(self.extras, key):
+            val = getattr(self.extras, key)
+            return _resolve(val, self) if resolve else val
+        return default
+
+    def __getattr__(self, name: str) -> Union[str, "LazyData"]:
+        """Access extra values as if they were members.
+
+        This makes members in the `extra` dataclass feel like
+        members of the `AbstractTableEntry` instance. Thus, pseudo
+        adding members is easy, and table generation is simplified.
+        """
+        extras = object.__getattribute__(self, "extras")
+        try:
+            return getattr(extras, name)
+        except AttributeError as err:
+            raise AttributeError(f"'{type(self)} nor {type(self.extras).__name__}' have field {name}") from err
+
+    def with_(self, **kw):
+        return evolve(self, **kw)
+
+
+# For Parameters:
+AsteriskColumn = ColumnSpec(
+    key="asterisk", header="", justify="left", width=1, style="red bold", converters=asterisk_required_converter
+)
+NameColumn = ColumnSpec(
+    key="name",
+    header="",
+    justify="left",
+    style="cyan",
+    formatter=wrap_formatter,
+    converters=[stretch_name_converter, combine_long_short_converter],
+)
+DescriptionColumn = ColumnSpec(key="description", header="", justify="left", overflow="fold")
+# For Commands:
+CommandColumn = ColumnSpec(
+    key="name",
+    header="",
+    justify="left",
+    style="cyan",
+    formatter=wrap_formatter,
+    converters=[stretch_name_converter, combine_long_short_converter],
+)
+
+
 @define
-class HelpPanel:
+class AbstractRichHelpPanel:
+    """Adjust the Format for the help panel!."""
+
+    from rich.box import ROUNDED, Box
+    from rich.console import Group as RichGroup
+    from rich.console import NewLine, RenderableType
+    from rich.panel import Panel
+    from rich.table import Column, Table
+    from rich.text import Text
+
     format: Literal["command", "parameter"]
```

Comment:
It was kind of a bodge before to have this `format` field, but I think you've organized the code enough now that it makes even less sense. I think it might make sense to have 2 subclasses here (`CommandHelpPanel` and `ParameterHelpPanel`). I think this also helps resolve the below "TODO".